app/app.go

```diff
@@ -924,16 +914,22 @@ func NewDesmosApp(
 		}
 	}
 
+	app.ScopedIBCKeeper = scopedIBCKeeper
+	app.ScopedIBCTransferKeeper = scopedIBCTransferKeeper
+	app.ScopedProfilesKeeper = scopedProfilesKeeper
+	app.ScopedWasmKeeper = scopedWasmKeeper
+
 	if loadLatest {
 		if err := app.LoadLatestVersion(); err != nil {
 			tmos.Exit(err.Error())
 		}
-	}
+		ctx := app.BaseApp.NewUncachedContext(true, tmproto.Header{})
```

Comment:
What is this used for?

Reply:
It is used for `InitializePinnedCodes` function, which initializes wasmVM caching contracts codes. See:
https://github.com/CosmWasm/wasmd/blob/88e01a98ab8a87b98dc26c03715e6aef5c92781b/x/wasm/keeper/keeper.go#L885
It calls the VM to load contracts to its memory:
https://github.com/CosmWasm/wasmvm/blob/1809ec56a49f1259ad0d5e94d66f960f8365c935/lib.go#L78


Honestly, I thought it was missing to add this part in Desmos, since it is added in wasmd app here:
https://github.com/CosmWasm/wasmd/blob/v0.30.0/app/app.go#L795-L806